developer_setup.md

```diff
@@ -37,8 +37,17 @@
   sudo passwd postgres
   # ↑ will ask to choose a password
   sudo postgresql-setup initdb
+  # if the above usage of postgresql-setup fails with 'command not found', use the following command with the relevant postgres 9.x version - below an example for postgres 9.1:
```

Comment:
I think the distinction is whether or not your system is managed by init or systemd as described [here](https://docs.fedoraproject.org/en-US/Fedora/16/html/Release_Notes/sect-Release_Notes-Changes_for_Sysadmin.html#id2838704) rather than whether the command is found or RHEL vs Fedora

So this section can be simplified to:
### For systems using systemd units
`sudo postgresql-setup initdb`
### For systems using SysV init scripts
`sudo service <postgres service name> initdb`